packages/transformers/assemblyscript/src/ASTransformer.js

```diff
@@ -0,0 +1,28 @@
+// @flow strict-local
+
+import {Transformer} from '@parcel/plugin';
+import invariant from 'assert';
+
+import * as asc from 'assemblyscript/cli/asc';
+
+export default (new Transformer({
+  async transform({asset}) {
+    await asc.ready;
+
+    let code = await asset.getCode();
+    const {binary} = asc.compileString(code, {
+      // optimize: true,
```

Comment:
The user should be able to modify the options here

Reply:
Yes, that's what I meant with 

> We probably also need a config file for the assemblyscript compiler options